src/Controller/Api/UploadController.php

```diff
@@ -2,98 +2,113 @@
 
```

Comment:
Moved the existing FilesController on top of the old UploadController, didn't change anything